src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -124,8 +132,24 @@ const WorkspaceContainer = ({ namespace, name, breadcrumbs, topBarContent, title
         apps, galaxyDataDisks, workspace, refreshApps
       })
     ]),
-    showTabBar && h(WorkspaceTabs, { namespace, name, activeTab, refresh, workspace }),
-    div({ role: 'main', style: Style.elements.pageContentContainer }, [children])
+    showTabBar && h(WorkspaceTabs, { namespace, name, activeTab, refresh, workspace, deletingWorkspace, setDeletingWorkspace, cloningWorkspace, setCloningWorkspace, sharingWorkspace, setSharingWorkspace }),
+    div({ role: 'main', style: Style.elements.pageContentContainer },
+      // Display the context bar only if the analysis tab is visible, for now
+      (isAnalysisTabVisible() ?
```

Comment:
based on this https://broadworkbench.atlassian.net/browse/IA-2709, I think we also want to restrict this to just the AnalysisTab since we are the first to introduce it (my understanding of timeline is this will be released with the analysis tab before other tabs start to add functionality). I think we can use `activeTab` in this expression to restrict where it shows (not sure what exactly that var represents, my guess would be the string in the `name` [here](https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/workspaces/workspace/Analyses.js#L473).

Reply:
OK good catch.. I'm meeting with Joy in a bit, can confirm with her.